__tests__/app.test.js

```diff
@@ -251,7 +299,30 @@ describe('GET', () => {
                     );
                 });
             })
-        })    
+        })
+        test("Query filter by topic 'cats' ", () => {
+            return request(app)
+            .get('/api/articles?topic=cats')
+            .expect(200)
+            .then(({ body: { articles } }) => {
+                expect(articles).toBeInstanceOf(Array);
+                articles.forEach((article) => {
+                    expect(articles).toHaveLength(1)
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            article_id: expect.any(Number),
+                            title: expect.any(String),
+                            topic: 'cats',
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            count: expect.any(Number)
+                        })
+                    );
+                });
+            })
+        })
```

Comment:
We should have a test for when the topic exists but has no articles, and we should expect an empty array

Reply:
could you please expand how a topic can exist wtihout an article, since it is listed as not null in our database table